addons/account/models/account_move.py

```diff
@@ -3342,6 +3342,20 @@ def _get_reconciled_invoices_partials(self):
                 exchange_diff_moves.append(partial.exchange_move_id.id)
         return invoice_partials, exchange_diff_moves
 
+    def _reconcile_reversed_moves(self, moves, reverse_moves, move_reverse_cancel):
+        ''' Reconciles moves and reverse moves
```

Comment:
Your doc string is identical to the method name, it could be a bit more detailed :wink: